assets/js/base/components/country-input/country-input.tsx

```diff
@@ -13,7 +13,7 @@ import Combobox from '../combobox';
 import './style.scss';
 import type { CountryInputWithCountriesProps } from './CountryInputProps';
 
-const CountryInput = ( {
```

Comment:
We have already exported this component as the default export. Can you explain why do we need to export it here?

Reply:
Hey there! Thanks for the question, I understand it might look odd.

It's for two reasons:

1. https://github.com/storybookjs/storybook/issues/13408 default exports are not correctly read by react-docgen currently. I think there is also a paragraph in the docs that mentions that but couldn't find it right now. So, that's the main issue…
2. We were discussing to move away from `default` exports to named exports anyways.

However, I didn't want to remove the `default` export at this moment as it might create bugs elsewhere. As I am very new to the repo, I'm not 100% sure on what's covered by tests and what is not, but I know we don't really stop the build on TS errors, so I thought I'd err on the safe side by not removing the default export.

Reply:
@sunyatasattva Thanks for the explanation, that makes sense to me. Can you add a comment to the component to prevent confusion for other folks? The PR is LGTM now 🚢

Reply:
Good idea! I'm merging this for now, and I'll add something to explain that in a README file or something like that, to avoid having to add the same comment over and over in each component.